tools/fuse_convert.py

```
@@ -310,14 +310,34 @@ def meta(mod_folder):
     "zamu.confusingsupplements",
     "confusingsupplements",
+    "zamu.foryourconvenience",
+    "foryourconvenience",
     "fuse",
 }
 
 
+def is_core_legacy_requirement(package_id: object) -> bool:
+    value = str(package_id or "").strip()
+    lowered = value.lower()
+    while lowered.endswith(".fuse") or lowered.endswith(".rail"):
```

Ruff (0.16.1)

[warning] Call `endswith` once with a `tuple`

Merge into a single `endswith` call

(PIE810)